src/main/scala/com/fulcrumgenomics/umi/CallDuplexConsensusReads.scala

```diff
@@ -0,0 +1,115 @@
+    |have at least one observation of each strand, duplex consensus reads are assembled by combining the evidence
+    |from the two single strand consensus reads.
+    |
+    |Because of the nature of duplex sequencing, this tool does not support fragment reads - if found in the
+    |input they are ignored.  Similarly read pairs for which consensus reads cannot be generated for one or
+    |other read (R1 or R2) are omitted from the output.
+    |
+    |Consensus reads have a number of additional optional tags set in the resulting BAM file.  The tag names follow
+    |a pattern where the first letter (a, b or c) denotes that the tag applies to the first single strand consensus (a),
+    |second single-strand consensus (b) or the final duplex consensus (c).  The second letter is intended to capture
+    |the meaning of the tag (e.g. d=depth, m=min depth, e=errors/error-rate) and is upper case for values that are
+    |one per read and lower case for values that are one per base:
+    |
+    |The tags break down into
+    |those that are single-valued per read:
+    |    * consensus depth      [aD,bD,cD] (int): the maximum depth of raw reads at any point in the consensus reads
+    |    * consensus min depth  [aM,bM,cM] (int): the minimum depth of raw reads at any point in the consensus reads
+    |    * consensus error rate [aE,bE,cE] (float): the fraction of bases in raw reads disagreeing with the final consensus calls
+    |and those that have a value per base (duplex values are not generated, but can be generated by summing):
+    |    * consensus depth  [ad,bd] (short[]): the count of bases contributing to each single-strand consensus read at each position
+    |    * consensus errors [ae,be] (short[]): the count of bases from raw reads disagreeing with the final single-strand consensus base
+    |The per base depths and errors are capped at 32,767.
+  """,
+  group = ClpGroups.Umi)
+class CallDuplexConsensusReads
+(@arg(flag="i", doc="The input SAM or BAM file.") val input: PathToBam,
+ @arg(flag="o", doc="Output SAM or BAM file to write consensus reads.") val output: PathToBam,
+ @arg(flag="p", doc="The Prefix all consensus read names") val readNamePrefix: Option[String] = None,
+ @arg(flag="R", doc="The new read group ID for all the consensus reads.") val readGroupId: String = "A",
+ @arg(flag="1", doc="The Phred-scaled error rate for an error prior to the UMIs being integrated.") val errorRatePreUmi: PhredScore = DefaultErrorRatePreUmi,
+ @arg(flag="2", doc="The Phred-scaled error rate for an error post the UMIs have been integrated.") val errorRatePostUmi: PhredScore = DefaultErrorRatePostUmi,
+ @arg(flag="m", doc="Ignore bases in raw reads that have Q below this value.") val minInputBaseQuality: PhredScore = DefaultMinInputBaseQuality,
+ @arg(flag="S", doc="The sort order of the output, if None then the same as the input.") val sortOrder: Option[SortOrder] = Some(SortOrder.queryname)
```

Comment:
The thing is that since we unmap the reads, the next logical step is to proceed into mapping.  And for that I think you really want it in queryname, so I'd like to leave that as the default so one can't forget it.  I think queryname is important here a) because I'm not sure MergeBamAlignment will like query-grouped but not sorted data (even though it could work) and b) it's probably not good to send reads through alignment that are approximately coordinate sorted.